### ISSUE TITLE
"Always open in container" and "limit to designated sites" aren't separate.

### ISSUE THREAD

**Issue by al0fdf** [mozilla/multi-account-containers#2348]
Multi-Account Containers Version: 8.0.6
Firefox Version: Waterfox G4.1.1.1 (64-bit)
All other addons are disabled apart from "Container Tabs"

### Actual behavior
When limiting a container to designated sites, the only way to add a site to this list is to select "Always open in x container".

### Expected behavior
"Always open in x container" and "limit to designated sites" should be separate. eg. I want the "Google" container to be limited to docs.google.com, but I do not want docs.google.com to always open in the "Google" container when I want to submit a form without being signed in.

### Steps to reproduce
1. Open Container Tabs
2. Go to docs.google.com
3. Select Always open in x container
4. In settings of x container select "Limit to designated sites"
5. No way to turn off "Always open in x container", or add site to "designated sites" without using "always open in x container"

**Comment by mrgum** [comment]
I agree they are conceptually distinct features and should be implemented as such

**Comment by venteto** [comment]
This also relates to an [overlapping issue / feature request](https://github.com/mozilla/multi-account-containers/issues/2222#issuecomment-1353490196) I'd like to see
